package-lock.json

```diff
@@ -0,0 +1,2738 @@
+{
```

Comment:
Why do we have a package-lock in the base folder?

Reply:
eww whoops
killing

Reply:
removed